src/sdk/paymaster/provider/arka.ts

```diff
@@ -0,0 +1,194 @@
+import { ethers } from "ethers";
+import { PaymasterProvider } from "../interface";
+import { ErrorMessage } from "../ErrorMsg";
+
+export class ArkaPaymaster implements PaymasterProvider {
+  readonly url: string;
+  readonly apiKey: string;
+  readonly chainId: number;
+  readonly queryParams: string;
+
+  constructor(chainId: number, apiKey: string, paymasterUrl: string) {
+    this.url = paymasterUrl;
+    this.queryParams = `?apiKey=${apiKey}&chainId=${chainId}`;
+    this.apiKey = apiKey;
+    this.chainId = chainId;
+  }
+
+  async getTokenPaymasterAddress(tokenSym: string) {
+    let response = null;
+    const entryPointAddress = '0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789';
```

Comment:
We need to make Arka compatible for both EP0.6.0 and EP0.7.0, so rename this variable as entryPointAddressV6.